metaflow/task.py

```diff
@@ -300,6 +331,20 @@ def clone_only(self, step_name, run_id, task_id, clone_origin_task, retry_count)
                 ),
             ],
         )
+        # We write a single line to the log to give a better experience to the user
+        # (particularly in the UI)
+        output.save_logs(
+            TASK_LOG_SOURCE,
+            {
+                "stdout": BytesIO(
+                    bytes(
+                        "Task was cloned from %s/%s"
```

Comment:
Do we strictly need this?

Reply:
This modifies the resume-from-local CLI logs as well and now we will have two log lines that say the same thing in the user logs

Reply:
one is not persisted though. We can remove the other one if you want to remove one. It's not just the UI, it's also the client for the logs.

Reply:
@romain-intel in general we should avoid polluting the user's `stdout` and `stderr` with our logging noise unless it is something critical that they must see.

In this case the contents of `Cloning results of a previously run task` is already available in `task.metadata_dict['origin-task-id']` if you need the info, so I don't think we should output it in the persisted logs unnecessarily.

The ephemeral output on the console is different, since it serves the role of informing the user that something is happening since resuming might take a while.

Reply:
I am not married to it. If you like that better I'll remove it. From the UI's perspective, it's just non-obvious both during the running of the clone and after:
- during the running of the clone, nothing will be printed (to your point of "it may take a while" that may be confusing although in practice it does not take long).
- after the cloning, you have to know that something that says "No logs found" isn't actually an error or a sign that something went wrong in the case of a clone and you have to know where to look to determine if it is a clone.